index.js

```diff
@@ -0,0 +1,55 @@
+const tasks = [];
+//console.log(tasks);
+
+// 定義
+const taskInput = document.getElementById('task-input');
+//console.log(taskInput);
+const addButton = document.getElementById('add-button');
+//console.log(addButton);
+const taskList = document.getElementById('task-list');
+//console.log(taskList);
+
+
+// 関数
+// タスクを追加
+const addTask = () => {
+  const task = taskInput.value; //入力された値を取得
+  //console.log(task);
+  if (task) {
+    const newTask = createTaskObject(task); //タスクオブジェクトを作成
+    tasks.push(newTask);  //配列に追加
+    appendTaskToHTML(newTask); //HTMLに追加
+    taskInput.value = '';  //入力欄を空にする
+  };
+};
+
+// タスクオブジェクトの作成
+const createTaskObject = task => {
+  return {
+    id: tasks.length,
+    comment: task,
```

Comment:
細かいのですが、オブジェクトはkeyとvalueの命名が同じ場合に省略記法が使用できます。
引数名をcommentにすれば、以下のように記述できます！
```suggestion
    comment,
```

この省略形はとても良く使用されるので、覚えておいてください。